pkg/placement/volumes.go

```diff
@@ -94,7 +94,7 @@ func getVolumeMounts(serviceName string) []corev1.VolumeMount {
 		{
 			Name:      "config-data-merged",
 			MountPath: "/var/lib/kolla/config_files/config.json",
-			SubPath:   "placement-" + serviceName + "-config.json",
+			SubPath:   "placement-api-config.json",
```

Comment:
needs to be reverted as there are two kolla configs, one for the dbsync and one for the api